src/components/Main/index.tsx

```diff
@@ -20,26 +21,60 @@ type Word = {
 const Main = () => {
   const randomWord = () => Math.floor(Math.random() * data.length)
   const [word, setWord] = useState<Word>(data[randomWord()])
+  const [voice, setVoice] = useState<SpeechSynthesisVoice | null>(null)
+
+  useEffect(() => {
+    const voices = window.speechSynthesis?.getVoices()
+    const jpVoice = voices?.find((voice) => /ja-JP/.test(voice.lang))
+
+    setVoice(jpVoice || voices?.[0])
+  }, [])
+
+  const speak = (word: string) => {
+    const patchedWord = word?.split('-').join('')
+    const utterance = new SpeechSynthesisUtterance()
+
+    utterance.text = patchedWord
+    utterance.lang = 'ja-JP'
+    utterance.voice = voice
+    utterance.rate = 0.8
+
+    window.speechSynthesis.speak(utterance)
+  }
 
   return (
     <>
       <S.Block>
         <Card
-          image={word.img}
           hiragana={word.jp.wd}
+          image={word.img}
           kanji={word.jp.kj}
-          romanji={word.jp.rmj}
           meaning={word.mean}
+          romanji={word.jp.rmj}
         />
 
-        <S.Icon>
-          <RefreshOutline
-            title="Get new word"
-            onClick={() => setWord(data[randomWord()])}
-          />
-        </S.Icon>
+        <S.IconWrapper>
+          <S.Icon>
+            <VolumeUpOutline
+              onClick={() => speak(word.jp.rmj)}
```

Comment:
Using Hiragana, here would be:

```suggestion
              onClick={() => speak(word.jp.wd)}
```